.github/workflows/basic.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Install apt packages
         run: |
           sudo apt-get update
-          sudo apt-get install -y clang cmake
+          sudo apt-get install -y clang cmake libnuma-dev
```

Comment:
In the past, we discussed relying on HWLOC instead of `libnuma` because HWLOC is already used by other oneAPI components.

Reply:
Ok, got it. I was confused that we added libnuma dependency but have not added HWLOC as well.